app/models/order.rb

```diff
@@ -0,0 +1,4 @@
+class Order < ApplicationRecord
+    belongs_to :user
+    belongs_to :doughnut
```

Comment:
So for now we are using belongs_to because each order will have one doughnut, but later we will make it has_many?

Reply:
Yup. Once we start working on the Carts feature, I can edit this schema so that an Order entity can consist of multiple doughnuts.